### PR TITLE
[#895] Better oneOf validation messages for records

### PR DESCRIPTION
**Pull request by Bjwebb**
#895 

(Still needs updating to a lib-cove release once https://github.com/OpenDataServices/lib-cove/pull/31 is merged/released.)

### CODE REVIEW

**Comment by Bjwebb** [comment]
[Data Review Tool: Correct reporting of large blocks of JSON](https://trello.com/c/dXzzBjKM/48-data-review-tool-correct-reporting-of-large-blocks-of-json)

**Comment by Bjwebb** [comment]
Translation done. @odscjames This should be ready for final review now.

File: cove_360/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -542,15 +545,19 @@ msgstr ""
 "concedida entre <strong>%(start_date)s</strong> y "
 "<strong>%(end_date)s</strong>."
 
-#: cove_360/templates/cove_360/explore.html:54
+#: cove_360/templates/cove_360/explore.html:107
+msgid "The file was "
+msgstr ""
```
Comment (odscjames) [inline cove_360/locale/es/LC_MESSAGES/django.po]: Where did this come from with no Translation?

Comment (Bjwebb) [reply cove_360/locale/es/LC_MESSAGES/django.po]: This string is only in 360 CoVE, which we don't translate.

**Pull request merged**